### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by se-bastiaan**
### Summary

Adding templates for issues and pull requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+name: Bug report
+about: Something is not working as expected
+title: ''
+labels: 'bug, priority: medium'
+assignees: ''
+
+---
+
+### Describe the bug
+A clear and concise description of what the bug is.
```
Comment (joren485) [inline .github/ISSUE_TEMPLATE/bug-report.md:11]: Maybe add these explanations/descriptions as comments?

Comment (se-bastiaan) [reply .github/ISSUE_TEMPLATE/bug-report.md:11]: I did this on purpose. Now you have to replace them. We have comments in the GitLab templates and they usually kind of clutter the issue source and I looked into the larger projects to see what they are doing and seems to be more standard. Thus I think it works.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Our contact information
+    url: https://thalia.nu/association/committees/42/
+    about: For any questions not related to this repository
+  - name: Security issues
+    url: https://thalia.nu/contact
```
Comment (joren485) [inline .github/ISSUE_TEMPLATE/config.yml]: I don't know if https://thalia.nu/contact is the best contact information for security issues.

**Review by joren485 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (se-bastiaan) [reply .github/ISSUE_TEMPLATE/config.yml]: I have completely removed it

**Pull request merged**